### PR TITLE
fix node

### PR DESCRIPTION
**Pull request by cdietrich**
fixed node and gradle issues

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: build.gradle [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 	repositories {
 		jcenter()
+		maven {
+            url "https://plugins.gradle.org/m2/"
+		}
 	}
 	dependencies {
 		classpath 'org.xtext:xtext-gradle-plugin:1.0.15'
-		classpath 'com.moowork.gradle:gradle-node-plugin:0.13'
+		classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
```
Comment (franzbecker) [inline build.gradle]: That looks a bit strange. Why was it there in the first place? It's using the old plugin mechanism (necessary sometimes, but deprecated).

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Sun Dec 18 11:08:04 CET 2016
+#Thu Sep 21 16:25:16 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-bin.zip
```
Comment (franzbecker) [inline gradle/wrapper/gradle-wrapper.properties]: latest and greatest is 4.2 - does it work with it as well?

Comment (cdietrich) [reply gradle/wrapper/gradle-wrapper.properties]: xtext still uses 3.5 cause of tons of gradle 4x bugs

File: build.gradle (hunk 1 above)
Comment (cdietrich) [reply build.gradle]: no idea it was no longer working

**Pull request merged**